support/docker/production/Dockerfile.buster

```diff
@@ -35,6 +35,9 @@ RUN chown -R peertube:peertube /data /config
 ENV NODE_ENV production
 ENV NODE_CONFIG_DIR /config
 
+ENV HTTP_PROXY=$HTTP_PROXY
```

Comment:
We don't need this

Reply:
We removed the ENV HTTP_PROXY=$HTTP_PROXY from Dockerfile.buster